src/utils/post-types.ts

```diff
@@ -1,3 +1,4 @@
 export enum PostTypes {
-  Article = "ActivityStreamsArticle"
+  Article = "ActivityStreamsArticle",
+  Note = "ActivityStreamsArticle"
```

Comment:
Note is an article?

Reply:
Here the mapping is made which component is loaded in the _Posts.vue_ on line 28. _Articles_ and _notes_ have the same properties so they go to the same component. I need to rename the _ActivityStreamsArticle_ component to _TextPost_. Would you have a better suggestion for the component?

Reply:
Okay. let's leave it like this for now then. 👍